BaseStation/MapDownloadTest/Program.cs

```diff
@@ -22,22 +24,15 @@ class Program
     {
         static void Main(string[] args)
```

Comment:
This class has alot of duplicated code. Make sure to call the code in other files for the test program.